### ISSUE TITLE
eject key doesnt detected on Apple MacBook Pro 13" - A1278 - Mid 2012

### ISSUE THREAD

**Issue by BlackMage2** [free5lot/hid-apple-patched#89]
the function keys doesnt work on keyboard

**Comment by BlackMage2** [comment]
some other keys don't work either

**Comment by BlackMage2** [comment]
okay, it works now. 'options hid_apple fnmode=2' in /etc/modprobe.d/hid_apple.conf was the trick

**Comment by BlackMage2** [comment]
but keyboard backlight control doesn't work

**Comment by BlackMage2** [comment]
any idea how i can make that keyboard backlight works?

**Comment by BlackMage2** [comment]
okay, i have now installed pommed and most keys work

**Comment by BlackMage2** [comment]
but the eject cd key doesn't work, in xev when i press the eject key only 
`KeymapNotify event, serial 48, synthetic NO, window 0x0,
    keys:  2   0   0   0   0   0   0   0   0   0   0   0   0   0   0   0   
           0   0   0   0   0   0   0   0   0   0   0   0   0   0   0   0`
is written in xev

any idea how can i enable the eject cd function?

**Comment by free5lot** [comment]
> any idea how can i enable the eject cd function?

I have no device with `Eject CD` button, so maybe somebody who uses this feature can help you somehow.

Keyboard backlight should not be affected by this patched version, it manages rebinds and swaps only.

**Comment by BlackMage2** [comment]
correct, but when i install pommed, the keyboard backlight works. :D

Is there a way to find unknown keys? does xev output anything that helps?

what is 'KeymapNotify event, serial 48' ?

why is no KeyPress and KeyRelease event? like the other function keys?

**Comment by BlackMage2** [comment]
> > any idea how can i enable the eject cd function?
> 
> I have no device with `Eject CD` button, so maybe somebody who uses this feature can help you somehow.
> 
> Keyboard backlight should not be affected by this patched version, it manages rebinds and swaps only.

oh, that is bad, on my device are only confusing key signals detected. eject media key doesnt recognized
my eject cd is right from the f12-key(=volume up function key). All functions works, only eject cd key is not recognized..

but xev is not helpful there, nothing special keys were right detected

yes keyboard lights are not affected

upps, yes that was my error.  i have installed pommed and the service was not right configured. default the device was /dev/dvd but the right device is /dev/cdrom. but why is /dev/dvd default? bug report can be closed
